config/configfile.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2023 Kapeta Inc.
+// SPDX-License-Identifier: MIT
+
+package config
+
+import (
+	"encoding/json"
+	"os"
+)
+
+// ReadConfigFile reads the environment configuration file and returns the map
+func ReadConfigFile() (map[string]string, error) {
+	out := make(map[string]string)
+	kapetaConfigPath := os.Getenv("KAPETA_CONFIG_PATH")
+
+	if kapetaConfigPath == "" {
+		return out, nil
+	}
+
+	// Open the JSON file
+	file, err := os.Open(kapetaConfigPath)
+	if err != nil {
+		panic(err)
+	}
+	defer file.Close()
+
+	decoder := json.NewDecoder(file)
+
+	err = decoder.Decode(&out)
+
+	if err != nil {
+		return out, err
```

Comment:
Since we aren't checking for the error anyways, why not just panic.
we do that if the file doesn't exist we should also do it when the format is not what we expected. Then you can also remove the error in the return